#!/usr/bin/env python
PACKAGE = "image_cloud"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("subscribe_topic_pcl",			str_t,	 	0, "topic pcl", "")
gen.add("subscribe_topic_image",		str_t,	 	0, "topic image", "")
gen.add("subscribe_topic_image_info",	str_t,	 	0, "topic image_info", "")
gen.add("publish_topic_pcl",			str_t,		0, "remapped pcl topic", "" )
gen.add("publish_topic_image",			str_t,		0, "remapped image topic", "" )
gen.add("publish_topic_image_info",		str_t,		0, "remapped image info topic", "" )
gen.add("republish_pointcloud",			bool_t,		0, "republish pointcloud", True)
gen.add("queue_size",					int_t,		0, "message filter queue", 30, 1, 100 )

exit(gen.generate(PACKAGE, "image_cloud", "sync_remap"))